### DISCUSSION TITLE
Disparities in the G-Drive Folder

### DISCUSSION THREAD

**Discussion by Kirs10-Riley** [kbuzard/labs#43] [General]
# Missing documents 

After comparing Jorge's notes with the G-Drive contents these are the files that I noticed were missing from the G-Drive:

- **pdf2image** - output for pdf2Jpg.py
- **CA_Block_Data.shp** - inputs for countSim_speedUpr, CountSim_tester, and multiprocess_test2
- **CA_ZCTA_Data.shp** - inputs for countSim_speedUpr, CountSim_tester, and multiprocess_test2
- **CA_Labs_Data.shp** - inputs for countSim_speedUpr, CountSim_tester, and multiprocess_test2
- **Cal_Labs.shp** - input for field_org, stat_calc. Output for firm_struc and shapify.
- **Labs1998.csv** - input for Geo_coder, pngwork, and prep_Labs
- **OCR_output** - input for OCR
- **ScanData** - input for pdf2Jpg.py
- **1998DART32.pdf** - input for pdf2Jpg.py
- **Cattell-all.dta** - input for pngwork
- **BlockData** - input for USA_block_emp. This could possibly refer to an entire folder named BlockData.

**Comment by kbuzard** [comment]
> After comparing Jorge's notes with the G-Drive contents these are the files that I noticed were missing from the G-Drive:

Can you be super specific about what you mean? These are files that we should have on the G: drive but are not? Or that ARE on the G:drive but are not included in Jorge's OneNote description of the python scripts?

**Comment by Kirs10-Riley** [comment]
I wrote this in the pull-request but when I went through the G-drive folder one more time to add the .gz notes. I saw that I missed 4 of the "missing" documents:

- **pdf2image** - output for pdf2Jpg.py

- **OCR_output** - input for OCR

- **ScanData** - input for pdf2Jpg.py

- **Cattell-all.dta** - input for pngwork

These four are not found anywhere in the kbuzard-s18 folder. 

For OCR_output there is a folder called OCR_Output_1998 in T-Burk\PngData that could be what Antonio is referring to. The rest I am not sure.

**Reply by kbuzard** [reply]
@Kirs10-Riley I don't see a pull request on Github. Can you check to see what's going on? Once I can see that, I'll have better context to try to figure out what's going on with these four files.

**Reply by kbuzard** [reply]
@Kirs10-Riley I've finally been able to get through the revised document. It's in great shape! I fixed a few typos, got rid of one of the zip folders, and moved a few things around (especially at the beginning) so that it was easier for me to follow (this is not to say the way you organized things was bad; this is probably just the particular way my brain works).

There are two things I'd like to you do still in the file:
1. There are a couple of instances where there is the notation "(duplicate)" on a file; would you add details as to where the main file is?
2. For each of the duplicates you know of, will you check to make sure that they aren't referenced in any of the programs or documentation, and if not, replace the duplicate file with a shortcut to the original?
   - If the last modified date does NOT match, then don't delete. Open both versions and see if you can see any obvious differences. Let me know what you find.

For the four "missing" documents, here are my thoughts:
- pdf2image - output for pdf2Jpg.py: this is NOT an output. It's a python module that's being imported. @JorgeValde This likely means that you need to modify your notes--I'm guessing that's why Kirsten thought this was an output.

- ScanData - this is not an input, but a folder that is created to save the output into. I don't see the folder, and I don't see the output files anywhere. Again, Jorge, you'll need to modify this in your notes, and I guess we'll see what happens as you get all these scripts running and documented. Perhaps he never ended up using this output so deleted it and forgot to delete this program, or maybe you'll need to create these files--although if you didn't find files with names like "Page_1.jpg" anywhere, that would probably mean that this didn't get used.

- OCR_output - similar to ScanData. This is a folder that was meant to hold a file called *OCR_Results.txt*, and I don't see that file anywhere.

- Cattell-all.dta - there are two copies of this file: one in the ramosRivera folder, and another in PngData (which is the place it's called from in the program). Was this a mistake or typo?
